docs: improve readme and add REACT_APP_ORDER_BOOK_URLS (#3173)
cowprotocol/cowswap: README.md

yarn ui:test
```

# Sitemap

The sitemap can be found in <./public/sitemap.xml>

To update its content:

1. Edit the list of pages in <./src/sitemap.js>
2. Run `yarn sitemap`
3. Commit the changes to git

# Configuration

The app has some default configuration (in `.env` and `.env.prod` file), but it's highly encouraged to define your own `.env.local`.

> Make a copy of `.env` named `.env.local`, this will allow you to set your own configuration only in your local environment.

Reviewer: ```suggestion
Inside `apps/cowswap-frontend`, make a copy of `.env` named `.env.local`, this will allow you to set your own configuration only in your local environment.
```

## Supported networks (⚠️ Required)

You should set your own RPC endpoints by defining the following environment variables:

```ini
# Define your own RPC endpoints

> REACT_APP_NETWORK_URL_1=https://mainnet.infura.io/v3/{YOUR_INFURA_KEY}
> REACT_APP_NETWORK_URL_5=https://goerli.infura.io/v3/{YOUR_INFURA_KEY}
> REACT_APP_NETWORK_URL_100=https://rpc.gnosischain.com

Reviewer 2: Since you mention the INFURA key down below, I think these examples could be generic. What do you think?

```suggestion
REACT_APP_NETWORK_URL_1=https://...  
REACT_APP_NETWORK_URL_5=https://...
REACT_APP_NETWORK_URL_100=https://...
```

Author: I will rebase and merge into this other PR changes, so i do all at once

Author: Fixed

To set your own `AppData`, change `REACT_APP_FULL_APP_DATA_<environment>` environment variable. For more details, check out the environment file (<.env>)

## Orderbook API Endpoints

Fee quote requests and posting orders are sent to the Orderbook API. This API has the responsibility of collecting orders and
handing them to the solvers.

> The reference implementation of the API is [gp-v2-services](https://github.com/cowprotocol/services).

Reviewer 2: ```suggestion
The reference implementation of the API is [CoW Protocol Services](https://github.com/cowprotocol/services).
```

The API endpoint is configured using the environment variable `REACT_APP_ORDER_BOOK_URLS`.

Normally, you don't need to change this variable, as it has a default value, but it can be handy to test your own API.

```ini
REACT_APP_ORDER_BOOK_URLS='{"1":"https://api.cow.fi/mainnet","100":"https://api.cow.fi/xdai","5":"https://api.cow.fi/mainnet"}'
```

# Wallet Connect

> The app uses a Wallet Connect v1 bridge.

Reviewer 2: Not anymore. We have removed it, and this section should be removed as well

Author: fixed
